src/Cmfd.cpp

```diff
@@ -1311,39 +1166,24 @@ int Cmfd::getCellNext(int cell_id, int surface_id) {
 
 
 /**
- * @brief Return whether optically thick diffusion correction factor is in use.
- * @return Whether optically thick diffusion correction factor is in use.
+ * @brief Return whether Larsen's effective diffusion coefficient correction
+ *        factor is in use.
+ * @return Whether Larsen's effective diffusion coefficient correction factor
+ *         is in use.
  */
-bool Cmfd::isOpticallyThick() {
-  return _optically_thick;
+bool Cmfd::isLarsensEDCOn() {
+  return _larsens_edc_on;
 }
 
 
 /**
- * @brief Set whether optically thick diffusion correction factor is in use.
- * @param optically_thick Boolean indicating whether optically thick diffusion
- *        correction factor is in use.
+ * @brief Set whether Larsen's effective diffusion coefficient correction factor
+ *        is in use.
```

Comment:
You might add a detail comment block highlighting those situations in which the factor should be turned on/off, and whether it is up to the user to turn it on/off or if CMFD does it automatically (it's not yet clear to me from reading the code).